docker/puppetserver-standalone/docker-entrypoint.d/90-consul.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+  CONSUL_HOSTNAME="${CONSUL_HOSTNAME:-consul}"
+  CONSUL_PORT="${CONSUL_PORT:-8500}"
+
+  cat <<SERVICEDEF > /puppet-service.json
+{
+  "name": "puppet",
+  "id": "$HOSTNAME",
+  "port": 8140,
+  "address": "$ipaddress",
+  "checks": [
+    {
+      "http": "https://${HOSTNAME}:8140/${PUPPET_HEALTHCHECK_ENVIRONMENT}/status/test",
+      "tls_skip_verify": true,
+      "interval": "30s",
+      "deregister_critical_service_after": "5m"
```

Comment:
The corresponding puppetdb PR uses a value of `10m` for this.

The difference is fine, just wondering if that was intentional?

Was that to account for the longer startup time of puppetdb (since it may have to do all those database migrations)?

Reply:
yeah, the difference was mostly since puppetdb was going to spin waiting for puppetserver to start before it started to start itself